lib/TestSuiteRunner.ts

```diff
@@ -1,15 +1,26 @@
-import {literal, namedNode, triple} from "@rdfjs/data-model";
+import { literal, namedNode, triple } from "@rdfjs/data-model";
 import * as LogSymbols from "log-symbols";
 import * as RDF from "rdf-js";
-import {IManifest} from "./IManifest";
-import {ManifestLoader} from "./ManifestLoader";
-import {ITestCase} from "./testcase/ITestCase";
+import { IManifest } from "./IManifest";
+import { ManifestLoader } from "./ManifestLoader";
+import { ITestCase } from "./testcase/ITestCase";
 import WriteStream = NodeJS.WriteStream;
-import {Util} from "./Util";
+import { Util } from "./Util";
 // tslint:disable:no-var-requires
 const quad = require('rdf-quad');
 const streamifyArray = require('streamify-array');
 
+// tslint:disable-next-line: interface-over-type-literal
+export type ITestSuiteConfig = {
```

Comment:
Why not just use an interface?

Reply:
To avoid typo's in optional parameters with config. A type will give stronger type hints to your IDE, and will be stricter regarding compilation. You usually don't want to extend this type.

Reply:
> You usually don't want to extend this type.

I don't agree with that.

This is the description of the `interface-over-type-literal` rule:

> Interfaces are generally preferred over type literals because interfaces can be implemented, extended and merged.

IMO, this definitely applies here as well, and I certainly see use cases where this may be needed.

Reply:
You're the author of the repo, so you decide ofcourse :smile: 
But consider this case: during config construction e.g. in the test cases, you write 'cachPath' or 'spec' instead of respectively 'cachePath' or 'specification'. The TS compiler will not be able to correct you. And even for mandatory arguments, it will not complain if you do something like `{...defaultConfig, output: outputFORMATtooverwrite}`. I do agree that in case an interface is needed, it should just become an interface.

Reply:
Sure, I get your point. The non-extensibility of types is both an advantage and disadvantage, depending on the circumstance.
Perhaps TS will introduce more flexible types in the future that will have the best of both worlds :-)